### PR TITLE
feat(auth): adopt per-flow PKCE cookies (authkit-session 0.5.1)

### PR DESCRIPTION
**Pull request by nicknisi**
## Summary

- Bump \`@workos/authkit-session\` to \`^0.5.0\`, which derives PKCE cookie names per-flow from sealed state (replaces the single \`wos-auth-verifier\` cookie).
- Thread \`state\` into \`clearPendingVerifier\` inside \`createHandleCallback\`'s \`bail()\`. When URL \`state\` is absent (malformed callback) the delete is skipped — there's no deterministic cookie name to target — and the 10-minute PKCE TTL cleans up orphans.
- Update tests to derive the expected cookie name from sealed state via \`getPKCECookieNameForState\`; add coverage for both the state-present and state-absent bail paths.
- README note: apply \`withAuth\` only to routes that serve top-level HTML documents. XHR endpoints that run through \`withAuth\` mint PKCE cookies that never complete the flow, which under per-flow naming can accumulate into HTTP 431 under concurrent load.

Version bump (0.3.0) will ship in a separate release PR.

## Test plan

- [ ] \`pnpm test\` — unit tests pass (handle-callback and get-sign-in-url suites cover per-flow name derivation and both bail paths).
- [ ] Manual: trigger a callback with a malformed \`?state=\` (or none) and confirm no \`Set-Cookie\` delete is emitted; with a valid \`?state=\` confirm a single per-flow verifier-delete is emitted.
- [ ] Manual: exercise the happy-path callback and confirm the session cookie is set and the verifier cookie is cleared.

### CODE REVIEW

File: src/tests/get-sign-in-url.test.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import { describe, it, expect, vi, beforeEach } from 'vitest';
 import type { Cookies, RequestEvent } from '@sveltejs/kit';
+import { getPKCECookieNameForState } from '@workos/authkit-session';
 import { createGetSignInUrl, createGetSignUpUrl } from '../server/auth.js';
 
 type AuthKitInstance = Parameters<typeof createGetSignInUrl>[0];
 
-const PKCE_COOKIE_NAME = 'wos-auth-verifier';
+const SEALED_VERIFIER = 'sealed-verifier';
+const EXPECTED_PKCE_COOKIE_NAME = getPKCECookieNameForState(SEALED_VERIFIER);
```
[the remark this answers is not in the document]
Comment (gjtorikian) [reply src/tests/get-sign-in-url.test.ts:11]: This looks right to me FWIW!

**Comment by nicknisi** [comment]
@devin-ai-integration Addressed #1 in 434797d — wrapped `clearPendingVerifier` in try/catch with a `console.warn`, falling back to the PKCE TTL on rejection. Also added two tests: one for the storage-rejection path (asserts the 302 still goes out and `Set-Cookie` is empty), and one for the try/catch branch bailing with no `state` (prevents a silent regression if the `if (state)` guard in `bail()` is inverted).

Skipping #2 — the JSDoc on `createHandleCallback` already documents the state-present vs state-absent split, and an inline note would duplicate it.

**Review by gjtorikian — APPROVED** [review]
